internal/proto/controller/storage/oplog/store/v1/oplog.proto

```diff
@@ -34,6 +34,14 @@ message Entry {
   // we are NOT storing this plain-text entry data in the db
   // @inject_tag: gorm:"-" wrapping:"pt,entry_data"
   bytes data = 8;
+
+  // the private id of the data key version used to encrypt the data.
+  // @inject_tag: gorm:"not_null"
+  string key_id = 9;
+
+  // the scope id associated with the key used to encrypt the data.
+  // @inject_tag: gorm:"not_null"
```

Comment:
```suggestion
  // @inject_tag: gorm:"default:null"
```
This should fix the problem with the tests. `gorm:"not_null"` causes gorm to set the value of `scope_id` to any empty string on insert. In Postgres, domain type constraints are checked before any triggers are run and '' is not allowed for `wt_scope_id` so the insert fails.